### ISSUE TITLE
Link together the check_* family

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/performance#68]
That is something we rarely use in the docstring, the "seealso" field. In that case, however, it could be useful to have all the `check_*` functions linked to one another, so that the user can discover and jump from function docs to function docs.

This could also be implemented via a paragraph in the details.

Aside from that, these functions are great and will necessitate a [blogpost](https://github.com/easystats/blog/issues/14) sooner or later ☺️

**Comment by strengejacke** [comment]
> Aside from that, these functions are great and will necessitate a blogpost (and a vignette?) sooner or later 

True, I wish I would have more time to write down at least a blog post... I think _performance_ is a package of high usefuleness for many users.

**Comment by DominiqueMakowski** [comment]
patience you must have. It'll come ☺️  I think after performance is out and report at least fixed, I'll try to write some posts :)

**Comment by strengejacke** [comment]
You mean parameters, not performance?

**Comment by DominiqueMakowski** [comment]
parameters* yup but it's ok as you read my mind 🔮

**Comment by IndrajeetPatil** [comment]
> I think performance is a package of high usefuleness for many users, but doesn't have the popularity it probably deserve.

That aged [poorly](https://twitter.com/patilindrajeets/status/1372078952517410820?s=20) 😅 

Yes, we can cross-link documentation files with `@seealso` and `@family` tags.

**Comment by IndrajeetPatil** [comment]
Thankfully, this is already the case of the `pkgdown` website, but I guess a number of users might be discovering functions via manual and help pages. The `Rd` tags will definitely be helpful for them to discover other checks.

<img width="1036" alt="Screenshot 2021-07-02 at 16 39 04" src="https://user-images.githubusercontent.com/11330453/124290703-0c98ac00-db54-11eb-98fa-d5994a723f88.png">

**Comment by strengejacke** [comment]
How does `@family` work?

**Comment by strengejacke** [comment]
And what does it do in the docs?
